tools/perftests_ci/perftest/references/daint_default/local-priority-fifo.json

```diff
@@ -0,0 +1,121 @@
+{
+  "outputs": [
+    {
+      "name": "future overhead - create_thread_hierarchical - latch",
+      "executor": "no-executor",
+      "series": [
+        0.181103,
+        0.164999,
+        0.160817,
+        0.160382,
+        0.160797,
+        0.158612,
+        0.159756,
+        0.157871,
+        0.160242,
+        0.158903,
+        0.159874,
+        0.183125,
+        0.201295,
+        0.160541,
+        0.159134,
+        0.159,
+        0.158936,
+        0.159248,
+        0.160268,
+        0.160075,
+        0.160112,
+        0.159893,
+        0.160277,
+        0.197405,
+        0.159989,
+        0.16086,
+        0.159677,
+        0.158815,
+        0.160455,
+        0.159504,
+        0.159059,
+        0.160579,
+        0.159318,
+        0.159357,
+        0.160229,
+        0.159832,
+        0.159765,
+        0.158727,
+        0.160149,
+        0.208617,
+        0.16103,
+        0.159422,
+        0.157584,
+        0.159327,
+        0.160649,
+        0.204878,
+        0.159935,
+        0.157804,
+        0.177953,
+        0.159734,
+        0.160693,
+        0.158198,
+        0.183047,
+        0.182969,
+        0.162546,
+        0.162969,
+        0.165859,
+        0.164218,
+        0.205005,
+        0.160459,
+        0.162256,
+        0.160447,
+        0.162036,
+        0.160576,
+        0.162289,
+        0.159485,
+        0.211715,
+        0.160659,
+        0.161976,
+        0.158974,
+        0.162789,
+        0.160145,
+        0.161426,
+        0.159305,
+        0.161761,
+        0.159118,
+        0.208235,
+        0.159191,
+        0.161147,
+        0.162074,
+        0.166119,
+        0.162476,
+        0.164486,
+        0.158131,
+        0.16167,
+        0.15857,
+        0.161807,
+        0.158366,
+        0.162475,
+        0.15946,
+        0.161569,
+        0.15926,
+        0.162446,
+        0.159642,
+        0.162136,
+        0.159139,
+        0.161846,
+        0.159218,
+        0.206696,
+        0.158911
+      ]
+    }
+  ],
+  "HPX": {
+    "commit": "75ed4099f789683dcff28307400e1f6bac229427",
+    "datetime": "2021-06-03T14:26:11+00:00"
+  },
+  "environment": {
+    "hostname": "nid00121",
+    "clustername": "daint",
+    "compiler": "/apps/daint/SSL/HPX/packages/llvm-11.0.0/bin/clang++ 11.0.0",
+    "datetime": "2021-06-03T16:48:02.369715+02:00",
+    "envfile": null
+  }
+}
```

Comment:
By the way, I assume this is the output you get if you just run the future overhead perftest? I.e. that's all that's required to generate a new reference?

Reply:
yes exactly!